agent/tox.ini

```diff
@@ -1,30 +1,13 @@
 [tox]
-envlist = docker
-
-[testenv:docker]
-allowlist_externals=docker
-passenv =
-    PYTHON_VERSION
-    TOXENV
-setenv =
-    PYTHON_VERSION=3.13
-    TOXENV=check,py313
-deps =
-usedevelop = false
-depends =
-skip_install = true
-commands =
-    docker build --build-arg PYTHON_VERSION={env:PYTHON_VERSION} -t balrog_agent-{envname}-test -f Dockerfile.test .
-    docker run --rm -v {toxinidir}:/app -v balrog_agent-{envname}-tox:/app/.tox -e TOXENV={env:TOXENV} balrog_agent-{envname}-test
+envlist = py313
```

Comment:
Would love to stop using tox, especially if we're only testing against one Python version.. but that's definitely follow-up fodder.